### PR TITLE
Fix editor-styles

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Editor take up full width and remove default editor-styles-wrapper that interfere with hurumap ui styles

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

## Desktop Screenshots

localhost_8080_wp-admin_post php_post=1 action=edit(iPad Pro)

## Mobile Screenshots

localhost_8080_wp-admin_post php_post=1 action=edit(iPhone 6_7_8 Plus)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: wp-content/themes/hurumap/functions.php [hunk 1]
```diff
@@ -134,9 +154,3 @@ function register_acf_block_types()
 if (function_exists('acf_register_block_type')) {
     add_action('acf/init', 'register_acf_block_types');
 }
-function wpb_adding_scripts()
-{
-    wp_enqueue_script("pdfJS", "https://mozilla.github.io/pdf.js/build/pdf.js");
-}
-
-add_action('enqueue_block_editor_assets', 'wpb_adding_scripts');
```
Comment (KhadijaMahanga) [inline wp-content/themes/hurumap/functions.php:142]: this script is needed for document viewer on indicator block

**Review by KhadijaMahanga — APPROVED** [review]

File: wp-content/themes/hurumap/assets/css/editor-style.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/*
+Theme Name: HURUmap
+Description: Used to style the TinyMCE editor.
+*/
+
+.wp-block {
+    max-width: 100%;
+}
```
Comment (kilemensi) [inline wp-content/themes/hurumap/assets/css/editor-style.css]: \n

File: wp-content/themes/hurumap/assets/css/editor-blocks.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/*
+Theme Name: Hurumap
```
Comment (kilemensi) [inline wp-content/themes/hurumap/assets/css/editor-blocks.css]: HURUmap

File: wp-content/themes/hurumap/functions.php [hunk 4]
```diff
@@ -2,9 +2,29 @@
...
+add_action('enqueue_block_editor_assets', 'hurumap_block_editor_styles');
+function hurumap_block_editor_styles()
+{
+    // Block styles.
+    wp_enqueue_style('hurumap-block-editor-style', get_theme_file_uri('/assets/css/editor-blocks.css'), array(), '1.1');
+    wp_enqueue_script('remove-default-styles-wrapper-script', get_theme_file_uri('/assets/js/remove-default-styles-wrapper.js'));
+
     /**
-     * Setup
+     * Blocks pdf preview script
```
Comment (kilemensi) [inline wp-content/themes/hurumap/functions.php]: @KhadijaMahanga ?

**Review by kilemensi — APPROVED** [review]
:rocket: @karimkawambwa

**Pull request merged**